addons/mass_mailing_sms/i18n/mass_mailing_sms.pot

```diff
@@ -698,7 +721,7 @@ msgstr ""
 #. module: mass_mailing_sms
 #: code:addons/mass_mailing_sms/wizard/sms_composer.py:0
 #, python-format
-msgid "STOP SMS : %s"
+msgid "STOP SMS: "
```

Comment:
Technically we should probably not change that in stable, at least until 17.0 ... we loose translations, if people did any, and it is not really a critical bugfix.